src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss

```diff
@@ -2,10 +2,54 @@
 
 $toolbarBackgroundColor: $white;
 $shadowColor: $mischka;
+$buttonsContainerBackgroundColor: $alabaster;
+
+$buttonsContainerBorder: 1px $gallery solid;
 
 .toolbar {
     height: 60px;
     margin: 0;
     background: $toolbarBackgroundColor;
     box-shadow: 0 3px 2px $shadowColor;
 }
+
+.buttons {
+    display: inline-block;
+    border-left: $buttonsContainerBorder;
+    border-right: $buttonsContainerBorder;
+    vertical-align: top;
+    background-color: $buttonsContainerBackgroundColor;
+}
+
+.pull-left {
+    float: left;
+}
+
+.pull-right {
+    float: right;
+}
+
+.icons,
+.locale {
+    display: inline-block;
```

Comment:
I think it would be easier to understand if this line is in both `.icons` and `.locale` instead defined like this. The same way as you've done it with `.icon` (this way it is already inconsistent).